simulationwindservice/pom.xml

```diff
@@ -48,6 +48,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-amqp</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
```

Comment:
Waar is dit voor?

Reply:
bij de mockmvc kan je dit doen:
.andExpect(jsonPath("$[0].name", is("Production")));

zo kan je de body van de response lezen en kijken wat je verwacht terug te krijgen